include/alpaka/mem/buf/cpu/ConstBufCpu.hpp

```diff
@@ -0,0 +1,129 @@
+/* Copyright 2025 Anton Reinhard
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+#pragma once
+
+#include "alpaka/dev/DevCpu.hpp"
+#include "alpaka/dev/Traits.hpp"
+#include "alpaka/mem/alloc/AllocCpuAligned.hpp"
+#include "alpaka/mem/buf/Traits.hpp"
+#include "alpaka/platform/PlatformCpu.hpp"
+#include "alpaka/vec/Vec.hpp"
+
+#include <functional>
+#include <memory>
+#include <type_traits>
+#include <utility>
+
+namespace alpaka
+{
+    //! The constant CPU memory buffer.
+    template<typename TElem, typename TDim, typename TIdx>
+    class ConstBufCpu : public internal::ViewAccessOps<ConstBufCpu<TElem, TDim, TIdx>>
```

Comment:
Like @fwyzard suggest, maybe it is a better idea to inherit from `BufCpu` and only specialize the [GetPtrNative](https://github.com/AntonReinhard/alpaka/blob/59d22208e7caa1a5c779e66556e9e99f8463e798/include/alpaka/mem/buf/BufCpu.hpp#L153) trait to save code.